### PR TITLE
issue 1112 - Address various UI issues

### PR DESCRIPTION
**Pull request by jiportilla**
issue 1112 - Address various UI issues

### CODE REVIEW

**Comment by jiportilla** [comment]
Fixed ES locale files

**Comment by jiportilla** [comment]
Fixed polOrg in exchange/agbot.go

**Comment by jiportilla** [comment]
Fixed es/locale file

File: cli/exchange/agbot.go [hunk 1]
```diff
@@ -133,6 +133,8 @@ func AgbotAddBusinessPolicy(org, userPw, agbot, polOrg string) {
 	org, agbot = cliutils.TrimOrg(org, agbot)
 
 	input := exchange.ServedBusinessPolicy{BusinessPolOrg: polOrg, BusinessPol: "*", NodeOrg: polOrg}
+        i18n.GetMessagePrinter().Printf("Adding Business policy org %s' to agbot '%s' ...", polOrg, agbot)
```
Comment (dabooz) [inline cli/exchange/agbot.go]: "Adding Business policy org %s' to agbot '%s'. The agbot will start looking for nodes that are compatible with this policy."

File: cli/exchange/agbot.go [hunk 2]
```diff
@@ -147,5 +149,7 @@ func AgbotRemoveBusinessPolicy(org, userPw, agbot, PolOrg string) {
 	cliutils.SetWhetherUsingApiKey(userPw)
 	org, agbot = cliutils.TrimOrg(org, agbot)
 	polId := formServicedObjectId(PolOrg, "*", PolOrg)
+	i18n.GetMessagePrinter().Printf("Removing Business policy org %s' from agbot '%s' ...", PolOrg, agbot)
```
Comment (dabooz) [inline cli/exchange/agbot.go]: Removing Business policy org %s' to agbot '%s'. The agbot is no longer looking for nodes that are compatible with this policy."

File: cli/exchange/node.go [hunk 3]
```diff
@@ -246,6 +246,8 @@ func NodeAddPolicy(org string, credToUse string, node string, jsonFilePath strin
 	}
 
 	// add/replce node policy
+	msgPrinter.Printf("Updating Node policy ...")
```
Comment (dabooz) [inline cli/exchange/node.go]: "Updating Node policy and re-evaluating all agreements based on this policy. Existing agreements might be cancelled and re-negotiated."

File: cli/exchange/node.go [hunk 4]
```diff
@@ -285,6 +287,8 @@ func NodeUpdatePolicy(org, credToUse, node string, jsonfile string) {
 			cliutils.Fatal(cliutils.CLI_INPUT_ERROR, msgPrinter.Sprintf("Invalid property list %s: %v", attribute, err))
 		}
 		newPolicy.Properties = newProp
+		msgPrinter.Printf("Updating Node %s policy properties in the horizon exchange ...", node)
```
Comment (dabooz) [inline cli/exchange/node.go]: Similar change as above.

File: cli/exchange/node.go [hunk 5]
```diff
@@ -300,6 +304,8 @@ func NodeUpdatePolicy(org, credToUse, node string, jsonfile string) {
 			cliutils.Fatal(cliutils.CLI_INPUT_ERROR, msgPrinter.Sprintf("Invalid constraint expression %s: %v", attribute, err))
 		}
 		newPolicy.Constraints = newConstr
+		msgPrinter.Printf("Updating Node %s policy constraints in the horizon exchange ...", node)
```
Comment (dabooz) [inline cli/exchange/node.go]: similar change as above

File: cli/exchange/node.go [hunk 6]
```diff
@@ -326,6 +332,8 @@ func NodeRemovePolicy(org, credToUse, node string, force bool) {
 	}
 
 	// remove policy
+	msgPrinter.Printf("Removing Node policy ...")
```
Comment (dabooz) [inline cli/exchange/node.go]: "Removing Node policy and re-evaluating all agreements based on just the builtin node policy. Existing agreements might be cancelled and re-negotiated."

File: cli/exchange/service.go [hunk 7]
```diff
@@ -584,6 +584,8 @@ func ServiceAddPolicy(org string, credToUse string, service string, jsonFilePath
 	}
 
 	// add/replace service policy
+	msgPrinter.Printf("Updating Service policy …")
```
Comment (dabooz) [inline cli/exchange/service.go]: similar changes are needed here as well and below in the remove section.

File: cli/exchange/node.go (hunk 4 above)
Comment (jiportilla) [reply cli/exchange/node.go]: changes implemented

**Pull request merged**